onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -297,7 +297,7 @@ void RegisterConvActivationFusionRules(SelectorActionRegistry& registry) {
   const std::string msDomainConv = SelectorActionRegistry::OpVersionsMapKey("NhwcConv", kMSDomain);
   auto selector = std::make_unique<selectors::ConvActivationSelector>();
 
-  registry.RegisterSelectorAndAction(name, {{"Conv", {1, 11}}, {msInternalNHWCDomainConv, {11}}, {msDomainConv, {1}}},
+  registry.RegisterSelectorAndAction(name, {{"Conv", {1, 11}}, {msInternalNHWCDomainConv, {1, 11}}, {msDomainConv, {1}}},
```

Comment:
Non-owner review. Just curious that why change `{11}` to `{1, 11}`. Does it mean that you limit the fuse optimization to v1~v11? Will v11+ still have this optimization?  Is it right if we change `msInternalNHWCDomainConv` to `{1}` like `msDomainConv`?

I don't know why the original code is `{{"Conv", {1, 11}}, {msInternalNHWCDomainConv, {11}}, {msDomainConv, {1}}` instead of `{{"Conv", {1}}, {msInternalNHWCDomainConv, {1}}, {msDomainConv, {1}}`
@skottmckay Just want to double check with you whether the current change is correct.